GoogleDataTransport/GDTCORTests/Unit/GDTCOREventTest.m

```diff
@@ -74,4 +74,52 @@ - (void)testArchiving {
   XCTAssertEqual(decodedEvent.clockSnapshot.timezoneOffsetSeconds, timezoneOffsetSeconds);
 }
 
+/** Tests encoding and decoding. */
+- (void)testNSSecureCoding {
```

Comment:
This doesn't actually test encoding and decoding. See the test above -testArchiving for an example. This just tests setting variables.